aspnet/mvc/models/validation.rst

```diff
@@ -1,8 +1,230 @@
-.. include:: /../common/stub-topic.txt
+Model Validation 
+================
+By `Rachel Appel <http://github.com/rachelappel>`_
 
-|stub-icon| Model Validation
-============================
+In this article:
 
-.. include:: /../common/stub-notice.txt
+.. contents:: Sections
+  :local:
+  :depth: 1
 
-.. _issue: https://github.com/aspnet/Docs/issues/116
+Introduction to model validation
+--------------------------------
+Before an app stores data in a database, the app must validate the data. Data must be scrubbed for security, verified that it is appropriately formatted by type and size, and it must conform to your rules. Validation is necessary although it can be redundant and tedious to implement. In MVC, validation happens on both the client and server. 
+
+Fortunately, .NET has abstracted validation into data annotations. These attributes contain validation code, thereby reducing the amount of code you must write. 
```

Comment:
- This says nothing about what MVC contributes. Need something like "And MVC executes the validation data annotations against submitted data as well as maps annotations to client-side validations."
- Not a fan of the term "data annotations" without further qualification. `[Display(Description = "Lovely property")]` is an example of a data annotation that has no impact at all on MVC's validation. We're talking specifically about validation attributes and, later, `IValidatableObject` and `IClientModelValidator` implementations. Could perhaps say "validation data annotations" as I did in the above bullet but I see "validation attributes" more often. @danroth27 what do you prefer?